deploy/charts/kubernetes-cluster-prep/values.yaml

```diff
@@ -0,0 +1,53 @@
+# Default values for conjur-cluster-prep.
```

Comment:
I made a bunch of changes to get the upcoming changes for unittest and schema validation to work.
Maybe easiest to copy this file:
https://github.com/diverdane/conjur-cluster-prep/blob/master/values.yaml

Reply:
Mind adding this script?:
https://github.com/diverdane/conjur-cluster-prep/blob/master/test-lint
(This might help you check for errors as you make changes.)

Reply:
If you do add the `test-lint` script, you'll also have to add the `tests/test-cert.pem` file.